webapp/portlet/src/main/webapp/activity-composer-app/extension.js

```diff
@@ -9,6 +10,16 @@ export function getActivityComposerExtensions() {
   return activityComposerActions;
 }
 
+export function getActivityComposerSwitchToAnArticle(messageLength) {
```

Comment:
The term 'switchToAnArticle' should be removed everywhere and replaced by something related to hint action. For example here it could be 'getActivityComposerHintActionExtensions'.
And we should probably rename the method `getActivityComposerExtensions` to `getActivityComposerActionExtensions`.